meta-mender-core/recipes-bsp/systemd-mender-config/systemd-mender-config.bb

```diff
@@ -0,0 +1,30 @@
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/${LICENSE};md5=89aea4e17d99a7cacdbeed46a0096b10"
+
+RDEPENDS_${PN} = "python3"
+
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
+SRC_URI = " \
+    file://ab_setup.py \
+    "
+
+S = "${WORKDIR}"
+
+do_compile() {
+    :
+}
+
+do_install() {
+    install -d ${D}${sbindir}
+    install -m 0755 "${WORKDIR}/ab_setup.py" "${D}${sbindir}"
+
+    # these names are required for dunfell compatibility
+    ln -s "../..${sbindir}/ab_setup.py" "${D}${sbindir}/fw_printenv"
+    ln -s "../..${sbindir}/ab_setup.py" "${D}${sbindir}/fw_setenv"
+
+    # these names will be used after dunfell and should be preferred
```

Comment:
I'm leaning towards removing the dunfell part, since I really don't want these potentially conflicting names to remain here. Just add the names in the middle [here](https://github.com/mendersoftware/mender/blob/b481af5576d6648dc8211fa84bf4d0d3f110a955/installer/bootenv.go#L59-L65) and it should work without this. New versions of the client will be cherry-picked to old branches, including both dunfell and zeus.